copis/gui/panels/controller.py

```diff
@@ -142,29 +184,38 @@ def _add_jog_controls(self) -> None:
         for col in (0, 1, 2, 3):
             xyzpt_grid.AddGrowableCol(col)
 
-        arrow_nw_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_nw', 20), size=(24, 24))
-        arrow_ne_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_ne', 20), size=(24, 24))
-        arrow_sw_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_sw', 20), size=(24, 24))
-        arrow_se_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_se', 20), size=(24, 24))
-
-        x_pos_btn = wx.Button(jog_sizer.StaticBox, label='X+', size=(24, 24))
-        x_neg_btn = wx.Button(jog_sizer.StaticBox, label='X-', size=(24, 24))
-        y_pos_btn = wx.Button(jog_sizer.StaticBox, label='Y+', size=(24, 24))
-        y_neg_btn = wx.Button(jog_sizer.StaticBox, label='Y-', size=(24, 24))
-        xy_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('keyboard', 24), size=(24, 24))
+        arrow_nw_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_nw', 20),
+            size=(24, 24), name='nw')
+        arrow_ne_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_ne', 20),
+        size=(24, 24), name='ne')
+        arrow_sw_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_sw', 20), size=(24, 24),
+        name='sw')
+        arrow_se_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_se', 20), size=(24, 24),
+        name='se')
```

Comment:
If you're going to have it take two lines, please indent the second one and keep them all consistent and readable. E.g.
```python
        arrow_nw_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_nw', 20),
            size=(24, 24), name='nw')
        arrow_ne_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_ne', 20),
            size=(24, 24), name='ne')
        arrow_sw_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_sw', 20),
            size=(24, 24), name='sw')
        arrow_se_btn = wx.BitmapButton(jog_sizer.StaticBox, bitmap=create_scaled_bitmap('arrow_se', 20),
            size=(24, 24), name='se')
```

Reply:
That was the plan. I don't know what happened with the indentation